python/shark_turbine/dynamo/tensor.py

```diff
@@ -49,15 +66,19 @@ class TurbineMode(TorchFunctionMode):
     """
 
     IMPLEMENTATIONS = {}
+    COMPUTE_METHODS = set()
+    CACHED_IMPLEMENTATIONS = {}
 
-    def __torch_function__(self, func, types, args=(), kwargs=None):
+    def __torch_function__(self, func, types, args=(), kwargs={}):
```

Comment:
Generally, don't do this: a dictionary is mutable and is not a great thing to store in a default arg... very easy to accidentally mutate and then that applies to the next call. Check if it is None at the top of the function and set it to `{}` if you have to. Since performance critical code, better to do as below and `({} if kwargs is None else kwargs)`. Generally do the `is None` check vs bool cast since the bool cast is expensive and returns True for an empty dict, resulting in two dict creations.

Reply:
Understood + done :)